src/builder-main.c

```diff
@@ -430,7 +430,7 @@ main (int    argc,
   g_type_ensure (G_TYPE_SOCKET_PROTOCOL);
   g_type_ensure (G_TYPE_SOCKET_ADDRESS);
 
-  orig_argv = g_memdup (argv, sizeof (char *) * argc);
+  orig_argv = g_memdup2 (argv, sizeof (char *) * argc);
```

Comment:
We can do this without bumping any requirements, however you need to update the `libglnx` submodule and use its header.